### PR TITLE
1.3.7 put org address

### PR DESCRIPTION
**Pull request by dangdennis**
## Description of the change

> Description here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]()

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: openapi.json [hunk 1]
```diff
@@ -1935,6 +1935,22 @@
           }
         }
       }
+    },
+    "/organization/address": {
```
Comment (kpamaran) [inline openapi.json:1939]: It feels awkward to have a PUT operation limited to a subset of fields. Why not a general `PUT /organization`?

File: schemas/UpdateOrgAddressInput.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      "example": "Suite 101"
+    },
+    "address_city": {
+      "type": "string",
+      "description": "Address city.",
+      "example": "Atlanta"
+    },
+    "address_state": {
+      "type": "string",
+      "enum": [
```
Comment (kpamaran) [inline schemas/UpdateOrgAddressInput.json:22]: Highly unlikely but I wonder if US territories like PR are valid inputs as well

File: requestBodies/UpdateOrgAddress.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "description": "Update the organization's address.",
```
Comment (ps2-controller) [inline requestBodies/UpdateOrgAddress.json:2]: Maybe add a note here like "this is the business address of your company. It is used for various administrative purposes within Canopy - for instance, it appears on borrowers' PDF-generated statements. "

**Review by ps2-controller — APPROVED** [review]

File: openapi.json (hunk 1 above)
Comment (dangdennis) [reply openapi.json:1939]: https://canopy-corp.slack.com/archives/C01608B94E8/p1626964082275500 I had this discussion with anurag. what do you think?

Comment (dangdennis) [reply openapi.json:1939]: I wanted a general PUT. so good to continue this discussion

Comment (kpamaran) [reply openapi.json:1939]: Then again this should be such a rare operation for an org that we can easily change/discard this.

Comment (dangdennis) [reply openapi.json:1939]: The main pro here is that our API can grow flexibly. Easier to relax things and combine things as opposed to bringing them all together

File: schemas/UpdateOrgAddressInput.json (hunk 2 above)
Comment (dangdennis) [reply schemas/UpdateOrgAddressInput.json:22]: <.< they should be yeah

File: openapi.json (hunk 1 above)
Comment (dangdennis) [reply openapi.json:1939]: @ps2-controller

**Review by kpamaran — APPROVED** [review]

**Pull request merged**